### DISCUSSION TITLE
Streamer files are not generated anywhere

### DISCUSSION THREAD

**Discussion by me0wg4ming** [RakambdaOrg/ChannelPointsMiner#439] [Q&A]
I tinkered a bit with the configration and i wanted the configration to load/save streamer  files but also load the follower list.

`{
	"accounts": [
		{
			"username": "samplenamer",
			"loginMethod": {
				"type": "tv",
				"authenticationFolder": "authentication/tv"
			},
			"streamerConfigDirectories": [
				{
					"path": "streamers",
					"recursive": true
				}
			],
			"loadFollows": true
		}
	]
`}``

But for some reason the Streamer files are never created (even if i only use the minimal streamer config without my modification.
Not sure what im doing wrong?

**Comment by Rakambda** [comment]
They are not meant to be created automatically. It's either you configure it with follows + default settings, or/and you define a file for adding some streamers manually/override settings.

**Comment by me0wg4ming** [comment]
How or where can i configure a follows + default string? Is it directly in the config file that i load from config.json? I seeked for an example but found none sadly. Thanks for the answer in advance.

**Reply by Rakambda** [reply]
[Default config is in the global config.](https://rakambda.github.io/ChannelPointsMiner/miner/main/configuration/global.html)
[Streamer settings is a file named by the streamer name inside one of the streamers directories.](https://rakambda.github.io/ChannelPointsMiner/miner/main/configuration/streamer.html)
